src/Tools/Microsoft.dotnet-openapi/test/OpenApiRefreshTests.cs

```diff
@@ -15,24 +10,24 @@ public class OpenApiRefreshTests : OpenApiTestBase
     {
         public OpenApiRefreshTests(ITestOutputHelper output) : base(output) { }
 
-        [Fact(Skip = "https://github.com/dotnet/aspnetcore/issues/32686")]
+        [Fact]
         public async Task OpenApi_Refresh_Basic()
         {
             CreateBasicProject(withOpenApi: false);
 
+            // Add <OpenApiReference/> to the project. Ignore initial filename.json content.
             var app = GetApplication();
             var run = app.Execute(new[] { "add", "url", FakeOpenApiUrl });
 
             AssertNoErrors(run);
 
+            // File will grow after the refresh.
             var expectedJsonPath = Path.Combine(_tempDir.Root, "filename.json");
-            var json = await File.ReadAllTextAsync(expectedJsonPath);
-            json += "trash";
-            await File.WriteAllTextAsync(expectedJsonPath, json);
+            await File.WriteAllTextAsync(expectedJsonPath, "trash");
```

Comment:
Does this mean it is safe to override the contents of `expectedJsonPath` instead of appending?

Reply:
The file is only read to check it's SHA in this test. Put another way, the intent is to demonstrate downloading a _longer_ JSON file than the original. New tests demonstrate identical and shorter downloads.

Reply:
(The `dotnet openapi` tool does not itself parse OpenAPI files. That comes later, when the changed project is built.)